src/main/java/io/stargate/sgv2/jsonapi/service/processor/CommandProcessor.java

```diff
@@ -102,7 +105,11 @@ public <T extends Command, U extends SchemaObject> Uni<CommandResult> processCom
 
                     // yet more mucking about with suppliers everywhere :(
                     yield (Supplier<CommandResult>)
-                        () -> new CommandResult(List.of(errorBuilder.apply(apiException)));
+                        () ->
```

Comment:
OK, so this is the cut point for collection path.
See another comment, when returned documents are empty, the response body is just
```
{}.
```
Is this intended?